src/CMakeLists.txt

```diff
@@ -1,13 +1,38 @@
-# This is hacky :(
-file(GLOB_RECURSE bustub_sources ${PROJECT_SOURCE_DIR}/src/*/*.cpp ${PROJECT_SOURCE_DIR}/src/*/*/*.cpp)
```

Comment:
I was worried that removing this line would cause any new files in the refsol not to be built properly.  Does this PR work with the refsols?

Reply:
There’s another PR in bustub private. We will also update CMakeLists if built with refsol.